app/controllers/admin/badges_controller.rb

```diff
@@ -44,11 +44,13 @@ def mass_award
     raise Discourse::InvalidParameters if csv_file.try(:tempfile).nil? || badge.nil?
 
     batch_number = 1
+    lineno = 1
```

Comment:
To be consistent with `batch_number`, I would have used `line_number` instead of ` lineno`.